components/bl-credit-card-component/component.json

```diff
@@ -0,0 +1,245 @@
+      "handlerId": "directionLogic",
+      "options": [
+        {
+          "value": "row",
+          "label": "Row"
+        },
+        {
+          "value": "row-reverse",
+          "label": "Row reverse"
+        },
+        {
+          "value": "column",
+          "label": "Column"
+        },
+        {
+          "value": "column-reverse",
+          "label": "Column reverse"
+        }
+      ]
+    },
+    {
+      "type": "checkbox",
+      "name": "labelsVisibility",
+      "label": "Labels Visibility",
+      "showInSettings": true,
+      "defaultValue": true,
+      "hasLogicHandler": false,
+      "handlerId": "labelsVisibilityLogic"
+    },
+    {
+      "type": "text",
+      "name": "submitButtonLabel",
+      "label": "Submit Button Label",
+      "showInSettings": true,
+      "defaultValue": "Pay",
+      "hasLogicHandler": false,
+      "handlerId": "submitButtonLabelLogic"
+    },
+    {
+      "type": "text",
+      "name": "cardNumberFieldPlaceholder",
+      "label": "Card Number Field Placeholder",
+      "showInSettings": true,
+      "defaultValue": "Card Number",
+      "hasLogicHandler": false,
+      "handlerId": "cardNumberFieldPlaceholderLogic"
+    },
+    {
+      "type": "text",
+      "name": "cardholderNameFieldPlaceholder",
+      "label": "Cardholder Name Field Placeholder",
+      "showInSettings": true,
+      "defaultValue": "Cardholder Name",
+      "hasLogicHandler": false,
+      "handlerId": "cardholderNameFieldPlaceholderLogic"
+    },
+    {
+      "type": "text",
+      "name": "expiryFieldPlaceholder",
+      "label": "Expiry Field Placeholder",
+      "showInSettings": true,
+      "defaultValue": "MM / YY",
+      "hasLogicHandler": false,
+      "handlerId": "expiryFieldPlaceholderLogic"
+    },
+    {
+      "type": "text",
+      "name": "cvcFieldPlaceholder",
+      "label": "CVC Field Placeholder",
+      "showInSettings": true,
+      "defaultValue": "CVC",
+      "hasLogicHandler": false,
+      "handlerId": "cvcFieldPlaceholderLogic"
+    },
+    {
+      "type": "text",
+      "name": "borderWidth",
+      "label": "Border Width",
+      "showInSettings": true,
+      "hasLogicHandler": false,
+      "handlerId": "borderWidthLogic",
+      "defaultValue": "1px"
+    },
+    {
+      "type": "select",
+      "name": "borderStyle",
+      "label": "Border Style",
+      "showInSettings": true,
+      "hasLogicHandler": false,
+      "handlerId": "borderStyleLogic",
+      "options": [
+        {
+          "value": "none",
+          "label": "none"
+        },
+        {
+          "value": "solid",
+          "label": "solid"
+        },
+        {
+          "value": "dotted",
+          "label": "dotted"
+        },
+        {
+          "value": "dashed",
+          "label": "dashed"
+        },
+        {
+          "value": "double",
+          "label": "double"
+        }
+      ],
+      "defaultValue": "solid"
+    },
+    {
+      "type": "color",
```

Comment:
I would remove the property since we've got the ability to change it using styles